android/app/src/main/java/ru/spbstu/amd/learnbraille/screens/practice/PracticeFragment.kt

```diff
@@ -71,7 +91,10 @@ class PracticeFragment : Fragment() {
                 return@Observer
             }
 
-            Toast.makeText(context, "Correct!", Toast.LENGTH_SHORT).show()
+            var toastMessage = "Правильно!"

```

Comment:
All user-visible string literals should be placed into resources.

Reply:
okay, placed into resources (note that it wasn't before and is like here in `dev`)